run_celery.py

```diff
@@ -10,13 +10,14 @@
 from app.aws.xray.context import NotifyContext
 
 environment = os.environ.get("NOTIFY_ENVIRONMENT")
-newrelic.agent.initialize(environment=environment)  # noqa: E402
 
 # notify_celery is referenced from manifest_delivery_base.yml, and cannot be removed
 from app import create_app, notify_celery  # noqa
 
 load_dotenv()
 
+newrelic.agent.initialize("newrelic.ini")
```

Comment:
Let's keep the environment parameter please, as it was previously on deleted line #13.